src/lib/accessibility/Behaviors/Menu/MenuItemBehavior.ts

```diff
@@ -1,5 +1,14 @@
 import { Accessibility } from '../../interfaces'
 
+/**
+ * @description
+ * The MenuItemBehavior is done for structure of menu item which consist of root element and anchor inside the root element.
+ * The MenuItemBehavior adds :
+ * - role 'presentation' on the root element
+ * - role 'menuitem' on anchor element
+ * - 'aria-expanded' attribute on anchor element based on "submenuOpened" property
```

Comment:
I assume, add attribute ```to```, not ```on```. Please, correct me, if I am wrong

Reply:
fixed :)